english/challenge-marshmellow-to-rock-off/castle-route/narnia.md

```diff
@@ -0,0 +1,26 @@
+You forget about the existence of that white light. Someone did say "don't go towards the light." 
+In your frantic scramble, you rush to the door and reach for the handle. But it won't budge. 
+You push and pull and it doesn't budge. The color drains from your face as you smell the sweet, 
+sweet savory goodness of burning marshmallows edging closer and closer towards you. 
+You turn around and your stomach does a backflip as you witness the slow but steady crawl of melted white goo towards you. 
+
+In a last ditch effort, you use all your might and try to break down the door but to no avail... when suddenly you hit upon it. 
+The door had no handle. You slide the door with your fork as a lever and it opens smooth as butter. 
+You use up your remaining strength to get inside and hold the door. After what seems like an hour, 
+but is actually only a couple of minutes, you can hear the marshmallow being retreating from the door and you breathe a sigh of relief.
+
+On your side of the door however, there is but a single torch and you can barely make out anything beyond a few metres away from you. 
+As you grope in the darkness, you stumble across a rusty sword. You pick it up and sheathe it, feeling a little stronger now that you have a weapon. 
+You pick up the torch and start to explore aroud you. It seems like you are in a large hallway as you can hear your own voice 
+resonate throughout the area. After going around this eerie hallway unlit, and lifeless (you begin to question the solitary torch there.), 
+you come across some stairs and a ladder that seems to lead above in the darkness somewhere. 
+
+As you ponder this, a shadowy figure leaps out from behind you with an intent to kill. 
+As you dodge this ambush by sheer luck, you unsheathe your new weapon, ready to take your stand against this madness.. 
+and suddenly, your rusty sword starts glowing as bright as your torch. The shadowy fiend's eyes burn with disgust under 
+its dark hood as it scurries to the stairs and vanishes in the darkness. As you examine some cryptic language written 
+on the hilt of the sword, you need to decide.
+
+(Go after the shadowy figure)[stairs-route/figure.md]
```

Comment:
Did you mean for this to be a link?  If so, your syntax is backwards.  The text should be in square brackets and the link in parentheses rather than vice versa.  You can view the page on GitHub on your fork to check if the link is displaying correctly.